api/src/services/files.ts

```diff
@@ -161,17 +161,22 @@ export class FilesService extends ItemsService {
 						xmp?: Record<string, unknown>;
 					} = {};
 					if (sharpMetadata.exif) {
-						const { image, thumbnail, interoperability, ...rest } = exif(sharpMetadata.exif);
-						if (image) {
-							fullMetadata.ifd0 = image;
-						}
-						if (thumbnail) {
-							fullMetadata.ifd1 = thumbnail;
-						}
-						if (interoperability) {
-							fullMetadata.interop = interoperability;
+						try {
+							const { image, thumbnail, interoperability, ...rest } = exif(sharpMetadata.exif);
+							if (image) {
+								fullMetadata.ifd0 = image;
+							}
+							if (thumbnail) {
+								fullMetadata.ifd1 = thumbnail;
+							}
+							if (interoperability) {
+								fullMetadata.interop = interoperability;
+							}
+							Object.assign(fullMetadata, rest);
+						} catch (err) {
+							logger.warn(`Couldn't extract EXIF metadata from file`);
+							logger.warn(err);
```

Comment:
The question here is if we consider Exif enough of a blocking problem to prevent the whole file upload. If the optional metadata read fails, is that important? The file can still be uploaded, used, and modified 🤔

Reply:
Oh  I realized it's not enough of a problem to block the whole process. Just logging it seems to be appropriate!